src/Sarif.Multitool/RewriteCommand.cs

```diff
@@ -49,14 +50,18 @@ public int Run(RewriteOptions rewriteOptions)
             return 0;
         }
 
-        private static RewriteOptions ValidateOptions(RewriteOptions rewriteOptions)
+        private bool ValidateOptions(RewriteOptions rewriteOptions)
         {
+            bool valid = true;
+
             if (rewriteOptions.Inline)
             {
                 rewriteOptions.Force = true;
```

Comment:
>rewriteOptions [](start = 16, length = 14)

Do you also have to set OutputFilePath = InputFilePath when Inline == true? ... or does Inline mean something different here?

Reply:
No, it means the same thing, and yes, it should be set. Existing bug, of course, now fixed.

---
In reply to: [312284417](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312284417) [](ancestors = 312284417)